.gitlab-ci.yml

```diff
@@ -75,6 +75,7 @@ build:
       - src/Examples/Scripts/Python
       - src/Examples/Algorithms/TrackFinding/share
       - src/Examples/Algorithms/Digitization/share
+      - src/pytest.ini
```

Comment:
Why does this go to the artifact? So you don't have to clone again? Don't you also need the `conftest.py`?

Reply:
Yeah without this file there are quite a lot of warnings from pytest about unkown marks. But you're right, cloning again is the better way, so I'll do that instead.